src/playlist-selectors.js

```diff
@@ -59,17 +86,10 @@ const stableSort = function(array, sortFn) {
  * exactly zero if the two are equal.
  */
 export const comparePlaylistBandwidth = function(left, right) {
-  let leftBandwidth;
-  let rightBandwidth;
+  let leftBandwidth = getBandwidth(left, false) || window.Number.MAX_VALUE;
+  let rightBandwidth = getBandwidth(right, false) || window.Number.MAX_VALUE;
 
-  if (left.attributes && left.attributes.BANDWIDTH) {
-    leftBandwidth = left.attributes.BANDWIDTH;
-  }
-  leftBandwidth = leftBandwidth || window.Number.MAX_VALUE;
-  if (right.attributes && right.attributes.BANDWIDTH) {
-    rightBandwidth = right.attributes.BANDWIDTH;
-  }
-  rightBandwidth = rightBandwidth || window.Number.MAX_VALUE;
+  videojs.log.warn('comparePlaylistBandwidth is deprecated');
```

Comment:
Should we add a suggestion on what alternative the user has if they get this message? e.g. one of our other deprecation messages says `player.hls is deprecated. Use player.tech_.hls instead.` I guess the alternative here is, write your own compare function, so maybe an alternative doesn't really apply here.